src/tox/helper/build_isolated.py

```diff
@@ -3,6 +3,9 @@
 dist_folder = sys.argv[1]
 backend_spec = sys.argv[2]
 backend_obj = sys.argv[3] if len(sys.argv) >= 4 else None
+backend_paths = sys.argv[4].split(":")
+
+sys.path[:0] = backend_paths
```

Comment:
wouldn't this be optional?

Reply:
`''.split(':')` returns `[]` and `sys.path[:0] = []` is noop

Reply:
quite cryptic behaviour instead of being explicit if you ask me

Reply:
I just wanted to keep it clean. If you think about it, it's pretty straightforward. The code basically says "inject any extra build paths in front of `sys.path`". And it shouldn't matter whether it's one path, two or none — the interface is the same.

Reply:
Might be cleaner but if you ask me it's more cryptic because you have to think about it, why it's correct rather than just be obvious. Not?